app/main.py

```diff
@@ -1,3 +1,18 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Callable:
+    cache_dict = {}
+
+    def wrapper(*args: Any, **kwargs: Any) -> Any:
+        key = str(args) + str(kwargs)
+        if key in cache_dict:
+            print("Getting from cache")
+            return cache_dict[key]
+        else:
+            result = func(*args, **kwargs)
+            cache_dict[key] = result
```

Comment:
Remove this

Reply:
> 

Hi, Olena:) Do you mean to delete a whole block with the condition "if..." or delete individual items in this block?